app/test_main.py

```diff
@@ -1,3 +1,41 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,human_age",
+    [
+        (0, 0, [0, 0]),
+        (14, 14, [0, 0]),
+        (15, 15, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17])
```

Comment:
Add test cases to check input when `cat_age` and `dog_age` have different values so that they are not dependent. Besides that, add test cases to test against negative numbers and incorrect types of data that raise the correct exception. you could separate the logic to test exceptions to a distinct function. Follow the checklist to make sure that all required cases are covered.